### ISSUE TITLE
Use a better favicon

### ISSUE THREAD

**Issue by doamatto** [mello-app/static-website#5]
The current favicon looks fine on dark theme (see image below), but looks poor due to colour contrast on light theme. This should be replaced with something more universal.

![Dark Theme](https://user-images.githubusercontent.com/30969346/94998277-81a76880-0565-11eb-86c0-a1a969e2b345.png)
![Light Theme](https://user-images.githubusercontent.com/30969346/94998286-8ec45780-0565-11eb-9488-348d8c905208.png)

I saw a bit that we can use to do this automagically on [some browsers](https://caniuse.com/#feat=link-icon-svg): ([source](https://catalin.red/svg-favicon-light-dark-theme/#light--dark-theme-detection-in-the-browser))
```
<svg width="50" height="50" viewBox="0 0 50 50" xmlns="http://www.w3.org/2000/svg">
  <style>
    rect {
      fill: green;
    }
    @media (prefers-color-scheme: dark) {
      rect {
        fill: red;
      }
    }
  </style>
  <rect width="50" height="50" rx="5"/>
</svg>
``` 

We'd have to still serve a fallback and have a CI setup to not have to manually edit the Figma export of the icon, if and when we change the favicon.

**Comment by doamatto** [comment]
- [ ] Generate both a light and dark favicon
- [ ] Serve based based on code-block
  - [ ] Serve a fallback method for changing the favicon
